packages/react-reconciler/src/ReactFiber.js

```diff
@@ -546,7 +546,6 @@ export function resetWorkInProgress(
     workInProgress.child = null;
     workInProgress.memoizedProps = null;
     workInProgress.memoizedState = null;
-    workInProgress.updateQueue = null;
```

Comment:
I think this is going to break other things that are likely not covered by tests. We’ve had many bugs related to not resetting enough.

Reply:
I'd have to dive deeper to be able to fix this then. I tried `initializeUpdateQueue(workInProgress)` instead which caused a bunch of "Unexpected pop" or "Unexpected Fiber popped". Also tried 
```
if (workInProgress.updateQueue !== null) {
  initializeUpdateQueue(workInProgress);
}
``` 
which caused the same issue

Reply:
Nevermind. VSCode has problems with flow and didn't realize `initializeUpdateQueue` was never defined because I imported from the same line as `import type`. The ReferenceError was just buried below 5 screens of console errors.